docs/alerts/monitors/alert-response.md

```diff
@@ -99,6 +99,9 @@ To resolve an alert, click a row to select it, then click **Resolve**.
 Note that the same threshold translating functionality supports to [Creating Monitor from the Metrics Explorer](/docs/alerts/monitors/create-monitor/#from-your-metrics-explorer) and [Opening Monitor in the Metrics Explorer](/docs/alerts/monitors/edit-settings/#view-in-metrics-explorer).
 :::
 
+{@import ../../reuse/alerts-timeslice.md}
```

Comment:
This doesn't really belong to alert response page. We can link it to either https://help.sumologic.com/docs/alerts/monitors/monitor-faq/ or https://help.sumologic.com/docs/alerts/monitors/create-monitor/#from-your-log-search.